h2o-r/h2o-package/R/explain.R

```diff
@@ -2371,7 +2389,8 @@ h2o.ice_plot <- function(model,
                          newdata,
                          column,
                          target = NULL,
-                         max_levels = 30) {
+                         max_levels = 30,
+                         binary_score_form = NULL) {
```

Comment:
Similar to python...
```suggestion
                         binary_score_form = c("probability", "logodds")) {
```

Reply:
I agree with this suggested change by @tomasfryda.

Reply:
[b127e35](https://github.com/h2oai/h2o-3/pull/6034/commits/b127e35e1145c6725d4c8527ee6d2e8397c35e03)